app/lib/search/index_simple.dart

```diff
@@ -141,6 +146,43 @@ class SimplePackageIndex implements PackageIndex {
     _isReady = true;
     _lastUpdated = new DateTime.now().toUtc();
   }
+
+  void _clearScores() {
+    _healthScores = null;
+    _popularityScores = null;
+  }
+
+  void _initScoresIfNeeded() {
+    if (_healthScores != null) return;
+    _healthScores = <String, double>{};
+    _popularityScores = <String, double>{};
+
+    final healthScorer =
+        new scoring.Summary(_documents.values.map((pd) => pd.health ?? 0.0));
+    final popularityScorer = new scoring.Summary(
+        _documents.values.map((pd) => pd.popularity ?? 0.0));
+    for (PackageDocument doc in _documents.values) {
+      if (doc.health != null) {
+        _healthScores[doc.url] = healthScorer.bezierScore(doc.health) * 100.0;
```

Comment:
What is a `bezierScore` and why can't we use **the already normalized** scores, which lie between `[-1.0, 1.0]`?

Either a package is healthy or not healthy, but it's health shouldn't depend on some complicated statistics of all other packages.